### PR TITLE
[Chore] Upgraded waypoint library to support react16

### PR DESCRIPTION
**Pull request by macrigiuseppe**
This pull request fixes sections of the website landing page using waypoint: https://github.com/uber/kepler.gl/issues/452

### CODE REVIEW

File: website/src/components/common/section.js [hunk 1]
```diff
@@ -82,7 +82,7 @@ class Section extends PureComponent {
     icon: PropTypes.string.isRequired,
     isDark: PropTypes.bool,
     background: PropTypes.string,
-    children: PropTypes.node.isRequired
+    // children: PropTypes.node.isRequired
```
Comment (chrisirhc) [inline website/src/components/common/section.js]: Should this be removed?

File: README.md [hunk 2]
```diff
@@ -1,4 +1,4 @@
-<p align="right">
+-wa<p align="right">
```
Comment (chrisirhc) [inline README.md]: What does this do?

File: website/src/components/common/section.js (hunk 1 above)
Comment (macrigiuseppe) [reply website/src/components/common/section.js]: i forgot to uncomment

File: README.md (hunk 2 above)
Comment (macrigiuseppe) [reply README.md]: that's a typo

Comment (macrigiuseppe) [reply README.md]: fixed

File: website/src/components/common/section.js (hunk 1 above)
Comment (macrigiuseppe) [reply website/src/components/common/section.js]: fixed

File: website/src/components/common/staggered-scroll-animation.js [hunk 3]
```diff
@@ -35,7 +35,7 @@ export default class StaggeredScrollAnimation extends PureComponent {
     duration: PropTypes.number,
     delay: PropTypes.number,
     animateOnce: PropTypes.bool,
-    Container: PropTypes.oneOfType([PropTypes.func, PropTypes.string]),
+    Container: PropTypes.oneOfType([PropTypes.object, PropTypes.func, PropTypes.string]),
```
Comment (chrisirhc) [inline website/src/components/common/staggered-scroll-animation.js]: Perhaps you're looking for `PropTypes.elementType`, going by the way it's being used: 
https://github.com/facebook/prop-types/commit/b67bbd40b5bf8b27475801ecdb7617071ceaccac

Comment (macrigiuseppe) [reply website/src/components/common/staggered-scroll-animation.js]: In this case staggered-scroll-animation will receive a styled-component element which is an object. See screenshot below

Screen Shot 2019-03-26 at 11 14 57 AM

But I will add PropTypes.elementType to the list of possible options

**Pull request merged**